README.md

```diff
@@ -51,10 +73,7 @@ Example of configuration of GitHub Connector can be found [here](https://github.
 
     Go to the main directory of the project in your workspace location and install the required dependencies:
 
+    ``` shell
+    cd github-connector
```

Comment:
```suggestion
    cd {MAIN_PROJECT_DIRECTORY}
```

or even better, how about you restructure that point like this? 
``````
3. Install dependencies in the main project directory. For example, for the GitHub Connector run:

  ``` shell
    cd github-connector
    dep ensure -vendor-only
  ```
``````